src/components/Collections/CollectionsIndexHeader.js

```diff
@@ -11,38 +11,63 @@ import { connect } from 'react-redux';
 
 import Colors from '../../constants/Colors';
 import CollectionsDialog, { COLLECTIONS_DIALOG_ACTIONS, CollectionsDialogText } from '../Dialog/CollectionsDialog';
-import { collectionsCounterSelector } from '../../redux/selectors';
+import { collectionsCounterSelector,creatingCollectionSelector  } from '../../redux/selectors';
 import HeaderCountIcon from '../Icons/HeaderCountIcon';
+import { isAddingNewCollection } from '../../redux/action-creators';
 
 const AddCollectionButton = ({ onPress }) => (
   <TouchableOpacity onPress={onPress}>
     <Feather name="plus-square" size={24} color={Colors.headerIcon} />
   </TouchableOpacity>
+
+);
+const AddCollectionInputButton = ({ onPress }) => (
+  <TouchableOpacity onPress={onPress}>
+    <Feather name="plus-square" size={24} color={Colors.headerIcon} />
+  </TouchableOpacity>
+
+);
+
+const SearchInputButton = ({ onPress }) => (
```

Comment:
You'll need to set `propTypes` for all new components. You can reference the `AddCollectionButton.propTypes` below.

Reply:
added PropTypes to all the variables missing PropTypes